packages/back-office-subscribers/nsip-representation/__tests__/index.test.js

```diff
@@ -33,81 +29,47 @@ const mockContext = {
 	}
 };
 const mockMessage = {
-	attachments: [
-		{
-			documentId: '8e706443-3404-4b89-9fda-280ab7fd6b68'
-		}
-	],
-	caseRef: 'BC0110001',
-	caseId: 151,
-	dateReceived: '2023-08-11T10:52:56.516Z',
-	examinationLibraryRef: '',
-	originalRepresentation: 'some rep text secret stuff',
-	redacted: true,
-	redactedBy: 'Bloggs, Joe',
-	redactedNotes: 'some notes here',
-	redactedRepresentation: 'some rep text',
-	referenceId: 'BC0110001-55',
+	representationId: 123,
+	caseRef: 'CASE-REF',
+	caseId: 123,
+	referenceId: 'reference-id',
+	status: 'published',
+	dateReceived: new Date('2023-01-01T09:00:00.000Z'),
+	redacted: false,
+	redactedRepresentation: 'redacted-representation',
+	originalRepresentation: 'original-representation',
+	representationFrom: 'PERSON',
+	representationType: 'Local Authorities',
 	registerFor: 'ORGANISATION',
-	representationFrom: 'AGENT',
-	representationId: 6409,
-	representationType: null,
-	representative: {
-		id: 6409,
-		contactMethod: 'email',
-		emailAddress: 'joe@example.com',
-		firstName: 'joe',
-		lastName: 'bloggs',
-		organisationName: 'agent company',
-		telephone: '01234 567891',
-		under18: false,
-		jobTitle: 'Engineer'
-	},
-	represented: {
-		id: 6410,
-		contactMethod: 'email',
-		emailAddress: 'jane@example.com',
-		firstName: 'jane',
-		lastName: 'bloggs',
-		telephone: '01234 567890',
-		under18: false
-	},
-	status: 'VALID'
+	representedId: 'represented-id',
+	representativeId: 'representative-id',
+	attachmentIds: ['123']
```

Comment:
Tiny, is it worth having 2 id's in the test to highlight the comma separated list of attachmentIds.